dm/openapi/spec/dm.yaml

```diff
@@ -583,7 +583,7 @@ paths:
           content:
             "application/json":
               schema:
-                $ref: "#/components/schemas/Task"
+                $ref: "#/components/schemas/OperateTaskResponse"
```

Comment:
I suddenly find that this pr would change the response of  `POST/PUT api/v1/tasks`, so WebUI need to adapt this change

/cc @zoubingwu